### DISCUSSION TITLE
Whisper error not working on my pc

### DISCUSSION THREAD

**Discussion by amrreda225** [openai/whisper#1541] [Q&A]
![image](https://github.com/openai/whisper/assets/81924062/982c917c-4650-4d2e-9d80-32ab76bd3238)

iam done with that 
what should i do please ??

**Comment by phineas-pta** [comment]
it needs internet to download models

**Reply by amrreda225** [reply]
of course i got a high speed internet 
how to download the modals ?

**Reply by amrreda225** [reply]
![image](https://github.com/openai/whisper/assets/81924062/18027c94-633f-4a8e-9ffb-610dffac55c5)

what CHANGED ? lol it works
